channels/migrations/0010_avatar_small_medium.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.10 on 2018-08-16 16:49
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import profiles.utils
+
+
+def nullify_avatars(apps, schema_editor):
+    """
+    Set all avatars to None. Since we don't show avatars yet
+    this avoids the work of populating avatar_small and avatar_medium
+    """
+    Channel = apps.get_model('channels', 'Channel')
+    # At the point the migration runs
+    Channel.objects.all().update(avatar_medium=None, avatar_small=None, avatar=None)
```

Comment:
What do you think of changing this to the following, just in case someone has uploaded a channel avatar (even though it doesn't display anywhere yet):
```python
for channel in Channel.objects.filter(avatar__isnull=False):
    channel.save(update_image=True)
```

Reply:
Since `Channel` is a [historical model](https://docs.djangoproject.com/en/2.1/topics/migrations/#historical-models) the `save` method won't be the same as the one we defined. However I can import the `make_thumbnail` method and use it here

Reply:
Never mind, it looks like there are no avatars on production yet anyway, so this is fine.